cypress/e2e/group3/metrics.ts

```diff
@@ -56,6 +56,17 @@ describe('Dockstore Metrics', () => {
     cy.get('[data-cy=metrics-partner-dropdown]').should('contain', 'AGC');
     cy.get('[data-cy=execution-metrics-total-executions-div]').should('contain', 4);
     cy.get('[data-cy=validations-table]').should('not.exist');
+    cy.get('[data-cy=metrics-execution-status-dropdown]').should('contain', 'All');
+    // Change execution status to Successful
+    cy.get('[data-cy=metrics-execution-status-dropdown]').click();
+    cy.get('[data-cy=metrics-execution-status-options]').contains('Successful').click();
+    cy.get('[data-cy=metrics-execution-status-dropdown]').should('contain', 'Successful');
+    cy.get('[data-cy=execution-metrics-table]').should('be.visible');
+    // Change execution status to FAILED_RUNTIME_INVALID
+    cy.get('[data-cy=metrics-execution-status-dropdown]').click();
+    cy.get('[data-cy=metrics-execution-status-options]').contains('Failed_runtime_invalid').click();
```

Comment:
In the execution status drop down HTML, I use the `titlecase` pipe so that single word statuses (SUCCESSFUL, FAILED, ABORTED) are displayed in a nicer way. Forgot that we had statuses with underscores...(see that line). 

IMO, I think this is fine for now because all of our execution status metrics right now are Successful, Failed, or Aborted so users won't see the underscore ones in the UI. I can create a ticket for 1.15.x to create a pipe to prettify the statuses with underscores, but for now, leaving as is unless others feel strongly about it

Reply:
Nvm, I created a pipe